inginious/common/tasks_problems.py

```diff
@@ -181,6 +181,15 @@ def input_is_consistent(self, task_input, default_allowed_extension, default_max
 
     @classmethod
     def parse_problem(self, problem_content):
+        # Checking problem edit inputs
+        if len(problem_content["offset"]) != 0:
```

Comment:
Here you will skip the check if `problem_content["offset"]` is an empty string, that is, if the field is left empty or as is. Empty string will override the default `1` value when loaded  in the `CodeProblem` constructor.

This will lead to an empty string being passed through CodeMirror and a first line beginning at `0`... you should either save the default value, or remove the empty string from the settings dict in this function.